distbench_test_sequencer.cc

```diff
@@ -163,9 +163,13 @@ grpc::Status TestSequencer::DoRunTestSequence(grpc::ServerContext* context,
     if (maybe_result.ok()) {
       auto &result = maybe_result.value();
       auto summary = SummarizeTestResult(result);
+      bool output_summary = false;
       for (const auto& s : summary) {
         maybe_result->add_log_summary(s);
-        LOG(INFO) << s;
+        if (s == "Instance summary:")
```

Comment:
This is a bit magic-value. Is there a less hacky way to supress the data we don't want to print? What data is it that we are skipping exactly?

Reply:
Change to remove the summary from the logs